Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM openjdk:11-jre as with_java
 
-FROM golang:1.20
+FROM golang:1.20.7
```

Comment:
does 7 matter? it would be built with the latest 1.20 golang on the build system, no?

Reply:
it doesn't, removed